openprescribing/frontend/context_processors.py

```diff
@@ -1,15 +1,17 @@
 import logging
+from urllib.parse import quote_plus
 
 from django.conf import settings
-from frontend.models import PCN
 
 logger = logging.getLogger(__name__)
 
 
 def support_email(request):
+    subject = f"OpenPrescribing Feedback ({request.get_full_path()}):"
     return {
         "SUPPORT_TO_EMAIL": settings.SUPPORT_TO_EMAIL,
         "SUPPORT_FROM_EMAIL": settings.SUPPORT_FROM_EMAIL,
+        "FEEDBACK_MAILTO": f"{settings.SUPPORT_TO_EMAIL}?subject={quote_plus(subject)}",
```

Comment:
thought: It's not really directly related to this PR, but if the support email that people send emails to is now a PHC one, and not `openprescribing.net`, I note that there are several mentions of contacting `feedback@openprescribing.net` in various JSON files.

Maybe these need updating, if they are in use?

https://github.com/search?q=repo%3Aebmdatalab%2Fopenprescribing+feedback%40openprescribing.net+language%3AJSON&type=code&l=JSON

![GitHub search results for the feedback email address in the openprescribing repository.](https://user-images.githubusercontent.com/3818079/223719947-19dd0637-ad32-419f-b8bc-1a7f38b8a96e.png)

Reply:
Also a good catch. I think those should be updated too.